docs/additional-functionality/rapids-shuffle.md

```diff
@@ -282,19 +282,24 @@ In this section, we are using a docker container built using the sample dockerfi
 
 ### Spark App Configuration
 
-1. Choose the version of the shuffle manager that matches your Spark version. 
+1. Choose the version of the shuffle manager that matches your Spark version.
    Currently we support:
 
-    | Spark Shim | spark.shuffle.manager value                              |
-    | -----------| -------------------------------------------------------- |
-    | 3.0.1      | com.nvidia.spark.rapids.spark301.RapidsShuffleManager    |
-    | 3.0.2      | com.nvidia.spark.rapids.spark302.RapidsShuffleManager    |
-    | 3.0.3      | com.nvidia.spark.rapids.spark303.RapidsShuffleManager    |
-    | 3.0.4      | com.nvidia.spark.rapids.spark304.RapidsShuffleManager    |
-    | 3.1.1      | com.nvidia.spark.rapids.spark311.RapidsShuffleManager    |
-    | 3.1.1 CDH  | com.nvidia.spark.rapids.spark311cdh.RapidsShuffleManager |
-    | 3.1.2      | com.nvidia.spark.rapids.spark312.RapidsShuffleManager    |
-    | 3.1.3      | com.nvidia.spark.rapids.spark313.RapidsShuffleManager    |
+    | Spark Shim    | spark.shuffle.manager value                              |
+    | --------------| -------------------------------------------------------- |
+    | 3.0.1         | com.nvidia.spark.rapids.spark301.RapidsShuffleManager    |
+    | 3.0.2         | com.nvidia.spark.rapids.spark302.RapidsShuffleManager    |
+    | 3.0.3         | com.nvidia.spark.rapids.spark303.RapidsShuffleManager    |
+    | 3.0.4         | com.nvidia.spark.rapids.spark304.RapidsShuffleManager    |
+    | 3.1.1         | com.nvidia.spark.rapids.spark311.RapidsShuffleManager    |
+    | 3.1.1 CDH     | com.nvidia.spark.rapids.spark311cdh.RapidsShuffleManager |
+    | 3.1.2         | com.nvidia.spark.rapids.spark312.RapidsShuffleManager    |
+    | 3.1.3         | com.nvidia.spark.rapids.spark313.RapidsShuffleManager    |
+    | 3.2.0         | com.nvidia.spark.rapids.spark320.RapidsShuffleManager    |
+    | 3.2.1         | com.nvidia.spark.rapids.spark321.RapidsShuffleManager    |
+    | 3.3.0         | com.nvidia.spark.rapids.spark330.RapidsShuffleManager    |
```

Comment:
oh, do we want Spark versions not released (SNAPSHOTS) in this table since our dist jar won't contain them?  They were there before so maybe we just fix for next release. @sameerz @abellina

Reply:
Yeah we should remove them.  So we are talking about: 3.0.4, 3.1.3, 3.2.1, and 3.3.0 correct?

Reply:
yes

Reply:
We can fix this in 22.02, as pointed out in issue #4319